transformer/src/main/java/org/eclipse/transformer/action/ActionType.java

```diff
@@ -21,7 +21,8 @@
    	ZIP, JAR, WAR, RAR, EAR,
 	JAVA,
         DIRECTORY,
-        PROPERTIES;
+        PROPERTIES,
+        TLD;
```

Comment:
Do we really need a new action for tld when a tld is just an xml file with a different file extension? @tbitonti

Reply:
See #33.

Reply:
@bjhargrave @tbitonti Thank-you, from what I understand with TextAction, I can simply remove the TLD Action, seems useless once TextAction is supported. Right?

Reply:
I think some configuration will be needed with the TextAction to process .tld files. I will let @tbitonti comment.